### PR TITLE
AMP-29286: Switch to administrative level names

### PR DESCRIPTION
**Pull request by vchihai**

### CODE REVIEW

File: amp/WEB-INF/src/org/digijava/module/aim/action/ExportCurrencyManager2XSL.java [hunk 1]
```diff
@@ -78,7 +78,8 @@ public ActionForward execute(ActionMapping mapping, ActionForm form,
        
        
        HSSFCell countryTitleCell = titleRow.createCell(cellIndex++);
-       HSSFRichTextString countryTitle = new HSSFRichTextString(TranslatorWorker.translateText("Country",locale,siteId));
+       HSSFRichTextString countryTitle = new HSSFRichTextString(
+               TranslatorWorker.translateText("Administrative Level 0", locale, siteId));
```
Comment (jdeanquin-dg) [inline amp/WEB-INF/src/org/digijava/module/aim/action/ExportCurrencyManager2XSL.java]: constant here too

File: amp/repository/aim/view/ar/filterPicker.jsp [hunk 2]
```diff
@@ -1,896 +1,896 @@
...
+			</table>
+			</td>
+			<td align="center">
+				<c:set var="tooltip_translation">
+					<digi:trn key="rep:filter:regionofinterest">Specify the region  of interest.</digi:trn>
+				</c:set>
+				<table align="center" cellpadding="1" cellspacing="1">
+						<tr bgcolor="#EEEEEE">
+					<td colspan="5" onmouseout="UnTip()" onmouseover="Tip('${tooltip_translation}');">
+					<b><digi:trn key="rep:filter:RegionTitle">Administrative Level 1</digi:trn></b>
```
Comment (jdeanquin-dg) [inline amp/repository/aim/view/ar/filterPicker.jsp]: you missed a translation in line 340

File: amp/repository/aim/view/editOrganisation.jsp [hunk 3]
```diff
@@ -1,2198 +1,2198 @@
...
-                                                                                <html:optionsCollection name="aimAddOrgForm" property="sectorScheme" value="ampSecSchemeId" label="secSchemeName" />
-                                                                            </logic:notEmpty>
-                                                                        </html:select>
-                                                                    </td>
-                                                                </tr>
-                                                                <tr>
-                                                                    <td style=" text-align:right" class="tdBoldClass">
-                                                                    	<digi:trn>Sector Preferences</digi:trn>
-                                                                    	<field:display name="Mandatory Indicator For Sector Preferences" feature="NGO Form">
-
```
Comment (jdeanquin-dg) [inline amp/repository/aim/view/editOrganisation.jsp]: line 1070 should be renamed to admin level 1 (FM entry) with its translation

File: amp/repository/aim/view/scripts/reportWizard/fundingGroups.js [hunk 4]
```diff
@@ -15,7 +15,7 @@ YAHOO.amp.reportwizard.fundingGroups["donor"]= new Array(
 				,'Donor Group'
 				,'Donor Agency'
 				,'Primary Sector'
```
Comment (jdeanquin-dg) [inline amp/repository/aim/view/scripts/reportWizard/fundingGroups.js]: I know it was hardcoded but lets take the chance to move admin levels to a constant

File: amp/repository/aim/view/editOrganisation.jsp (hunk 3 above)
Comment (jdeanquin-dg) [reply amp/repository/aim/view/editOrganisation.jsp]: Line 1078 change Specify Region to specify admin level 1 (and fix its translation)

**Comment by jdeanquin-dg** [comment]
CreateOrEditCategory has a lot of messages that needs to be changed

File: amp/repository/um/view/viewEditUser.jsp [hunk 5]
```diff
@@ -1,677 +1,677 @@
...
+	                                                                    <tr id="notificationEmailRow">
+	                                                                        <td width="169" align="right" height="30"style="font-size: 11px; font-weight: bold; color:#000;">
+	                                                                            <digi:trn>Notification Email</digi:trn>
+	                                                                            <font color="red">*</font>
+	                                                                        </td>
+	                                                                        <td width="190" height="30">
+	                                                                           <html:text name="umViewEditUserForm" property="notificationEmail" style="background-color: #FFFFFF;border: 1px solid #D0D0D0;color: #767676;font-size: 11px;margin: 5px;padding: 2px; width:180px;" styleId="notificationEmail"/>
+	                                                                        </td>
+	                                                                    </tr>
+																	<tr>
```
Comment (jdeanquin-dg) [inline amp/repository/um/view/viewEditUser.jsp]: line 154 has a message that needs to be translated

File: amp/repository/aim/view/scripts/reportWizard/fundingGroups.js (hunk 4 above)
Comment (vchihai) [reply amp/repository/aim/view/scripts/reportWizard/fundingGroups.js]: I don't think it is ok to put some literals hardcoded and some of them as constant. Or we put all of them in constants (that will be used only in this js file) or we don't put at all.

File: amp/repository/um/view/viewEditUser.jsp (hunk 5 above)
Comment (vchihai) [reply amp/repository/um/view/viewEditUser.jsp]: "Please Select Country" is a translatable message. 
On the other hand, the 'country' in this context is not related to adm levels. No action is needed here.

File: amp/repository/aim/view/editOrganisation.jsp (hunk 3 above)
Comment (vchihai) [reply amp/repository/aim/view/editOrganisation.jsp]: I think the idea of a regional organization is a little bit different from administrative levels.

**Pull request merged**